src/main/java/uk/gov/justice/hmpps/prison/repository/FinanceRepository.java

```diff
@@ -1,8 +1,21 @@
 package uk.gov.justice.hmpps.prison.repository;
 
+import org.springframework.data.jpa.repository.Query;
 import uk.gov.justice.hmpps.prison.api.model.Account;
 
+import java.math.BigDecimal;
+import java.util.Date;
+
 public interface FinanceRepository {
 
     Account getBalances(long bookingId, final String agencyId);
+
+    void insertIntoOffenderTrans(final String prisonId, final long offId, final long offBookId,
```

Comment:
I don't think `final` is part of the method signature and is therefore redundant?